addons/capetrader/capetrader.lua

```diff
@@ -0,0 +1,516 @@
+--[[Copyright © 2016, Lygre, Burntwaffle@Odin
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of CapeTrader nor the
+      names of its contributors may be used to endorse or promote products
+      derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL Lygre, Burntwaffle@Odin BE LIABLE FOR ANY
+DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.]]--
+
+_addon.name = 'CapeTrader'
+_addon.author = 'Lygre, Burntwaffle'
+_addon.version = '1.0.2'
+_addon.commands = {'capetrader', 'ct'}
+
+--NOTE: It is possible that the correct values for the following four variables can change after a version update.
+local gorpaID = 0x010F9099
+local mhauraID = 0x0F9
+local gorpaTargetIndex = 0x099
```

Comment:
While this helps outline the values, what I meant is to only define one of them. Then only one needs to be updated. For example, you can define only Gorpa's ID and derive the other two from it:
```lua
local gorpaID = 0x010F9099
local zoneID = math.floor(gorpaID / 0x100) % 0x1000
local gorpaIndex = gorpaID % 0x1000
```

Or you can define only the index and get Mhaura's ID from the resources:
```lua
local gorpaIndex = 0x099
local mhauraID = res.zones:with('english', 'Mhaura').id
local gorpaID = 0x01000000 + mhauraID * 0x1000 + gorpaIndex
```

Actually looking at it now it's not exactly prettier :D

The point being only one value should be variable when the others can be derived from it. This simply reduces the maintenance work and the surface area for manual bugs. But in this case the gains are admittedly rather small, so I'll leave that up to your judgment.